spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -575,12 +575,22 @@ def candidate_builds(app_id, version_id)
     #####################################################
 
     # @param (testing_type) internal or external
-    def build_trains(app_id, testing_type, platform: nil)
+    def build_trains(app_id, testing_type, platform: nil, tries: 5)
```

Comment:
Interesting, how does it relate to `with_reply`? I see that `with_tunes_retry` is only used once, however it seems like it is used together, since the actual request is always sent with `with_reply`, correct? @lacostej